src/main/java/com/commercetools/sync/commons/helpers/BaseSyncStatistics.java

```diff
@@ -0,0 +1,237 @@
+package com.commercetools.sync.commons.helpers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.netty.util.internal.StringUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.String.format;
+
+public class BaseSyncStatistics {
+    private static final Logger LOGGER = LoggerFactory.getLogger(BaseSyncStatistics.class);
+
+    private String reportMessage;
+    private int updated;
+    private int created;
+    private int failed;
+    private int processed;
+    private long startTime;
+    private long processingTimeInDays;
+    private long processingTimeInHours;
+    private long processingTimeInMinutes;
+    private long processingTimeInSeconds;
+    private long processingTimeInMillis;
```

Comment:
well, why instead of one variable we should keep and every time clean/update 4? This is at least more error prone.